storage-class/nfs/README.md

```
@@ -88,7 +92,18 @@
 
 
 ## Uninstall NFS Client Provisioner
-* Run `./delete-nfs-csi.sh` to uninstall `nfs-csi ` helm/chart.
+* Run `./delete-nfs-cient-csi.sh` to uninstall `nfs-csi ` helm/chart.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)